src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -288,67 +258,82 @@ class PopoverReportActionContextMenu extends React.Component {
-    }
-
-    render() {
-        return (
-            <>
-                <PopoverWithMeasuredContent
-                    isVisible={this.state.isPopoverVisible}
-                    onClose={this.hideContextMenu}
-                    onModalShow={this.runAndResetOnPopoverShow}
-                    onModalHide={this.runAndResetOnPopoverHide}
-                    anchorPosition={this.state.popoverAnchorPosition}
-                    animationIn="fadeIn"
-                    disableAnimation={false}
-                    animationOutTiming={1}
-                    shouldSetModalVisibility={false}
-                    fullscreen
-                    withoutOverlay
-                    anchorRef={this.anchorRef}
-                >
-                    <BaseReportActionContextMenu
-                        isVisible
-                        type={this.state.type}
-                        reportID={this.state.reportID}
-                        reportActionID={this.state.reportActionID}
-                        draftMessage={this.state.reportActionDraftMessage}
-                        selection={this.state.selection}
-                        isArchivedRoom={this.state.isArchivedRoom}
-                        isChronosReport={this.state.isChronosReport}
-                        isPinnedChat={this.state.isPinnedChat}
-                        isUnreadChat={this.state.isUnreadChat}
-                        anchor={this.contextMenuTargetNode}
-                        contentRef={this.contentRef}
-                        originalReportID={this.state.originalReportID}
-                    />
-                </PopoverWithMeasuredContent>
-                <ConfirmModal
-                    title={this.props.translate('reportActionContextMenu.deleteAction', {action: this.state.reportAction})}
-                    isVisible={this.state.isDeleteCommentConfirmModalVisible}
-                    shouldSetModalVisibility={this.state.shouldSetModalVisibilityForDeleteConfirmation}
-                    onConfirm={this.confirmDeleteAndHideModal}
-                    onCancel={this.hideDeleteModal}
-                    onModalHide={this.callbackWhenDeleteModalHide}
-                    prompt={this.props.translate('reportActionContextMenu.deleteConfirmation', {action: this.state.reportAction})}
-                    confirmText={this.props.translate('common.delete')}
-                    cancelText={this.props.translate('common.cancel')}
-                    danger
+    const showDeleteModal = (reportID, reportAction, shouldSetModalVisibility = true, onConfirm = () => {}, onCancel = () => {}) => {
+        onCancelDeleteModal.current = onCancel;
+        onComfirmDeleteModal.current = onConfirm;
+
+        reportIDRef.current = reportID;
+        reportActionRef.current = reportAction;
+
+        setShouldSetModalVisibilityForDeleteConfirmation(shouldSetModalVisibility);
+        setIsDeleteCommentConfirmModalVisible(true);
+    };
+
+    useImperativeHandle(ref, () => ({
```

Comment:
We missed to add `contentRef` which caused this issue - https://github.com/Expensify/App/issues/28538

Reply:
cc: @OlimpiaZurek